src/main_gui.cpp

```diff
@@ -235,10 +236,11 @@ enum {
 
 struct MainWindow : Window
 {
-	uint refresh;
+	GUITimer refresh = GUITimer(LINKGRAPH_REFRESH_PERIOD);
```

Comment:
kinda surprised this works, given `LINKGRAPH_REFRESH_PERIOD` is declared after this. Probably something to do with `static`. Maybe reorder?

Reply:
Turns out it was reinitialized in the constructor anyway... Regardless, I've moved initialization to the constructor in all methods, and added 'explicit' to prevent accidental constructor usage where the call to reset the timer was not updated properly (but was still 'correct' as the constructor does the same thing as SetInterval())